public/pages/Overview/components/Widgets/RecentFindingsWidget.tsx

```diff
@@ -83,14 +68,28 @@ export const RecentFindingsWidget: React.FC<RecentFindingsWidgetProps> = ({
 
   return (
     <WidgetContainer title={'Recent detection rule findings'} actions={actions}>
-      <TableWidget
-        columns={columns}
-        items={findingItems}
-        sorting={{ sort: { field: 'time', direction: SortDirection.DESC } }}
-        loading={loading}
-        message={widgetEmptyMessage}
-        className={widgetEmptyMessage ? 'sa-overview-widget-empty' : undefined}
-      />
+      {findingItems.length === 0 ? (
+        <EuiEmptyPrompt
+          style={{ position: 'relative' }}
+          body={
+            <div style={{ display: 'flex', justifyContent: 'center', padding: '32px' }}>
+              <p style={{ position: 'absolute', top: 'calc(50% - 20px)' }}>
+                <EuiText size="s">
+                  <span style={{ display: 'block' }}>No recent findings.</span>Adjust the time range
```

Comment:
why not use p or div instead of making a block span?

Reply:
Originally modeled it after what was done in TopRulesWidget to keep it consistent but changed it to use <p> in the helper function.